pkg/sentry/control/fs.go

```diff
@@ -36,10 +36,41 @@ type Fs struct {
 
 // TarRootfsUpperLayerOpts contains options for the TarRootfsUpperLayer RPC.
 type TarRootfsUpperLayerOpts struct {
+	// ContainerID identifies which container's rootfs upper layer should be
+	// serialized.
+	ContainerID string
 	// FilePayload contains the destination for output.
 	urpc.FilePayload
 }
 
+// Returns a referenced mount namespace for the given container ID,
+// or the init container if no ID is provided. Caller must DecRef the returned mntns when done.
```

Comment:
The term "init container" means something else in k8s context: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/.

When no ID is provided, we use `f.Kernel.GlobalInit().Leader().MountNamespace()`, which is the root container's mount namespace. So maybe use "root container" here.